financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,183 @@
+
+. Decrypt the AMS login credentials that go in through an Activity Parameter.
+. Run the AMS.
+. Log in to the AMS using the decrypted credentials.
+. Select the database using another in-going Activity Parameter.
++
+image::rpa-setup-activity-parameters-wizard.png[RPA Builder Activity Parameter Wizard]
+
+==== Minimum Steps to Adapt in This Workflow
+
+. Change the following in the xref:rpa-builder::toolbox-control-run-program.adoc[Run Program Action Step]:
+.. Replace the values in the *File name*, *Parameters*, and *Window Title* fields with the values from your AMS.
+.. Click the *Test* button to run the application so that you can use it to adapt the following Action Steps:
++
+image::rpa-setup-run-program-wizard.png[RPA Builder Run Program Wizard]
+
+. Use the *Start from scratch* option to capture and test a new pattern that indicates the login page of the AMS is visible onscreen:
++
+image::rpa-setup-image-search-wizard.png[RPA Builder Image Search Wizard]
+
+. Re-identify the field to enter the password. It may be necessary to add a similar Action Step previously for entering the user name in the xref:rpa-builder::toolbox-app-automation-keystrokes-to-app-element.adoc[Keystrokes to App Element Action Step].
++
+image::rpa-setup-keystrokes-app-element-wizard.png[RPA Builder Keystrokes to App Element Wizard]
+
+. Re-identify the *Login* button, which is used in the xref:rpa-builder::toolbox-app-automation-click-app-element.adoc[Click App Element Action Step]:
++
+image::rpa-setup-click-app-element-wizard.png[RPA Builder Click App Element Wizard]
+
+. Wait for the login process to complete by choosing a pattern that will disappear after completion:
++
+image::rpa-setup-choose-pattern.png[RPA Builder Image Search Wizard choose pattern]
+
+. Adapt the other Action Steps of the same types in a similar way. Or, add or delete Action Steps based on what is needed to log in to your AMS.
+
+=== Activity 2: Find a Record in the AMS
+
+The second activity finds a record in the AMS using a lookup code.
+
+. Enter the lookup code, which was provided as an Activity Parameter, as locate criterion to search for the account data set.
+. Click on the result entry to open the account data, and then wait until it is fully loaded.
+
+image::rpa-setup-lookup-code.png[RPA Builder lookup code]
+
+Change the following Action Steps:
+
+. Adapt the first Action Steps as you did in the previous Activity.
+. Adapt the scan area in the xref:rpa-builder::toolbox-text-recognition-ai-ocr-screen-based.adoc[AI OCR Action Step] in which RPA Bot searches for the record entry with the lookup code:
++
+image::rpa-setup-ai-ocr-wizard.png[RPA Builder AI OCR screen-based Wizard]
+
+. Adapt the rest of the Action Steps as you did in the previous Activity.
+
+=== Activity 3: Retrieve and Export Data from the AMS
+
+The third activity retrieves data and exports the data from the AMS so that it can be imported into to Salesforce Financial Services Cloud.
```

Comment:
```suggestion
The third activity retrieves data and exports the data from the AMS so that it can be imported to Salesforce Financial Services Cloud.
```